exploration/expression-attributes.md

```diff
@@ -88,10 +60,10 @@ At least the following expression attributes should be considered:
     whether the expression should or should not be localised.
     The values here correspond to those used for this property in HTML and elsewhere.
 
-    > Example embedding a non-translatable French literal in an English message:
+    > Example embedding a non-translatable CLI command in a message:
```

Comment:
Non-translatable is a not a use case.
`@translate=no` is unnecessary.

A function is either specified, or "implied" from the argument type.
I the example the function is not specified, and the type is string. So the implied function is `:string`, which does nothing.
And arguments are always non-localizable.

So `{|bonjour|}` is already unlocalizable.

---

The locale is also unnecessary, since there is no function to do any transformation, locale-aware or not.

For markup the locale might be mapped to a `lang` attribute in HTML, or `<lang>` in SSML (https://cloud.google.com/text-to-speech/docs/ssml#lang).
So I can see a use case for markup.

But I see no reason for this to be a "global" attribute, it would be specific to the markup dialect.
Same as "href", which is it is HTML specific.

---

I think we really have to separate runtime attributes from the l10n directives.